### PR TITLE
Remove unit test results from workflow and update the readme for the same

### PR DESCRIPTION
**Pull request by abhishekmishragithub**
## Description
ClickUp task: NA

What functionality does this add/problem does this address?
- Removes the unit test result from CI
Issue # The unit test result was being redundant as we are already using codecov.

This is a small change.

- [ ] This PR part of a larger effort
  - what will be addressed in separate PRs?

Have you added:
- [ ] TODO comments about the work not shipped in this PR
- [ ] TODO comments about nice to have refactoring
- [ ] doc strings and comments about design decisions and approach taken/not taken

## Checklist
- [x] Does the PR have an informative, carefully chosen **title**?
- [x] Does the branch you propose to merge have your name in it, `your-name/more-about-change` ?
- [x] Updated README.md and inline documentation as appropriate
- [ ] New files have copyright statement at the top and a careful, useful description of what the file does
- [x] Have you followed the other [Coding standards](https://github.com/predictionmachine/pm-coding-template#code-contents)?
- [x] Applied *labels* to the PR

### CODE REVIEW

File: .github/workflows/pm-gh-actions.yml [hunk 1]
```diff
@@ -227,9 +226,6 @@ jobs:
           python -m pip install --upgrade pip
           if [ requirements.txt ]; then pip install -r requirements.txt; fi
           if [ requirements-dev.txt ]; then pip install -r requirements-dev.txt; fi
-      - name: Test with pytest
```
Comment (tchklovski) [inline .github/workflows/pm-gh-actions.yml:230]: want to keep eg
```
        run: |
          python -m pytest --continue-on-collection-errors
```

**Review by tchklovski — CHANGES_REQUESTED** [review]
want to keep pytest pls

File: .github/workflows/pm-gh-actions.yml (hunk 1 above)
Comment (abhishekmishragithub) [reply .github/workflows/pm-gh-actions.yml:230]: Fixed

**Review by tchklovski — APPROVED** [review]
👍 lgtm!

**Pull request merged**